src/test/scala/org/kibanaLoadTest/helpers/CloudHttpClient.scala

```diff
@@ -322,9 +327,30 @@ class CloudHttpClient {
   def resetPassword(id: String): Map[String, String] = {
     logger.info(s"Reset password: deployment $id")
     val response = httpPost(
-      s"/$id/elasticsearch/main-elasticsearch/_reset-password"
+      s"/deployments/$id/elasticsearch/main-elasticsearch/_reset-password"
     )
     if (response.isEmpty) throw new RuntimeException(MSG_NO_RESPONSE)
     response.get.jsonString.parseJson.convertTo[Map[String, String]]
   }
+
+  def getDeployments: Map[String, String] = {
+    logger.info(s"Search for running deployments")
+    val response =
+      httpPost(
+        "/deployments/_search",
+        Helper.loadJsonString(searchDeploymentsTemplate)
+      )
+    val jsonString = response.get.jsonString
+    val names = jsonString
+      .extract[String](
+        Symbol("deployments") / elements / Symbol("name")
+      )
+      .toArray
+    val ids = jsonString
+      .extract[String](
+        Symbol("deployments") / elements / Symbol("id")
+      )
+      .toArray
+    Map() ++ (names zip ids)
```

Comment:
@wayneseymour you might like it :) This how you can convert 2 string arrays into single Map in Scala

Reply:
I love it!